<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewContactWindow</class>
 <widget class="QMainWindow" name="ViewContactWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(180, 180, 180);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>431</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sobrenome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>País</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Número</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(180, 180, 180);
	border-radius: 5px;
	padding: 2px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(55, 55, 55);
}
QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24);
}</string>
    </property>
    <property name="text">
     <string>Buscar Contatos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Contatos Salvos:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(180, 180, 180);
	border-radius: 5px;
	padding: 2px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(55, 55, 55);
}
QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24);
}</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
